streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/YarnQueueController.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.console.core.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@Validated
+@RestController
+@RequestMapping("flink/yarnQueue")
+public class YarnQueueController {}
```

Comment:
empty controller, no method ?

Reply:
Yes.  i plan to add the methods declaration and implementation in the  next mr.
Of course， I'd like to add its in this commit if needed in some rule.  
Please let me know what is your opinion.
Thank you very much.

Reply:
> Yes. i plan to add the methods declaration and implementation in the next mr. Of course， I'd like to add its in this commit if needed in some rule. Please let me know what is your opinion. Thank you very much.

This will make review difficult, not a complete feature implementation, it is difficult to give a review opinion

Reply:
thx for reply.
Got it & updated it.

Reply:
> thx for reply. Got it & updated it.

I'll review it as soon as possible